pkg/collector/reconcile/deployment_test.go

```diff
@@ -209,7 +209,8 @@ func TestExpectedDeployments(t *testing.T) {
 		assert.NoError(t, err)
 		assert.True(t, exists)
 		assert.Equal(t, orgUID, actual.OwnerReferences[0].UID)
-		assert.NotEqual(t, expectedTADeploy.Spec.Template.Spec.Containers[0].Image, actual.Spec.Template.Spec.Containers[0].Image)
+		assert.True(t, len(actual.Spec.Template.Spec.Containers[0].Args) == 1)
+		assert.True(t, actual.Spec.Template.Spec.Containers[0].Args[0] == "--enable-prometheus-cr-watcher")
```

Comment:
```suggestion
		assert.ElementsMatch(t, actual.Spec.Template.Spec.Containers[0].Args, []string{"--enable-prometheus-cr-watcher"})
```

This would simplify the test setup, in the case there are more options added.